library/spdm_crypt_lib/libspdm_crypt_cert.c

```diff
@@ -1080,6 +1081,22 @@ bool libspdm_x509_common_certificate_check(const uint8_t *cert, size_t cert_size
         goto cleanup;
     }
 
+    /* 3. Verify signature algorithm. */
```

Comment:
The other numbers need to be re-numbered, or maybe just removed entirely. Not sure how numbers help here.

Reply:
Thanks for your suggestions.  
Because my other PR change the file too, there will be some conflicts. 
I just make the smallest change in this PR.
I want to fix the number order after one PR merge.

Reply:
Ok.